src/java.base/share/classes/java/nio/file/Files.java

```diff
@@ -2920,7 +2921,7 @@ public static Path walkFileTree(Path start, FileVisitor<? super Path> visitor)
      *
      * @see #readAllLines
      */
-    public static BufferedReader newBufferedReader(Path path, Charset cs)
+    public static @MustCall("close") BufferedReader newBufferedReader(Path path, Charset cs)
```

Comment:
This shouldn't be necessary (`BufferedReader` is already `@MustCall("close")`).  I think you meant to add this annotation to `newDirectoryStream`, but as it turns out that already has a `@MustCall("close")` annotation.

Reply:
Thanks.  I fixed some other oversights too.